bluesky/run_engine.py

```diff
@@ -228,6 +228,18 @@ def __init__(self, md=None, *, loop=None, preprocessors=None,
         if md is None:
             md = {}
         self.md = md
+        self.md['versions'] = {}
```

Comment:
It may be better to use `md.setdefault('version', {})` which will not overwrite if the user passes something in like:

```python
RE = RunEngine({'version': {'my_other_lib': 5}})
```

```suggestion
        self.md.setdefault('versions', {})
```

```